src/mongofinil/core.clj

```diff
@@ -1,13 +1,14 @@
 (ns mongofinil.core
   "A Mongoid-like library that lets you focus on the important stuff"
-  (:require [somnium.congomongo :as congo]
-            [somnium.congomongo.coerce :as congo-coerce :refer [*translations*]]
-            [mongofinil.validation :as mv]
-            [mongofinil.validation-helpers :as mvh]
+  (:require [clojure.string :as str]
+
```

Comment:
Are these blank lines intentional?

Reply:
Yes, but they're an old habit from my Java days so I don't mind if we want to enforce a different-but-also-consistent style.